README.md

```diff
@@ -1,58 +1,70 @@
 
-* Set `export GOOGLE_APPLICATION_CREDENTIALS=/path/to/secret.json` in your
-shell, most likely in your `~/.bashrc`. Get the service account .json file,
-either from a collaborator or by logging into the [Service Accounts page](
+* *[Optional]* Run `./lint.sh --fix` to run clang-format and eslint on all relevant files and format in
+place. Clang-format will run automatically on commit and eslint will run automatically on push so
+this just saves you the extra commits. 
+
+## Running tests
+* A service account secret .json file is used to credential the test
+runner. Get the file either from a collaborator or by logging into the [Service Accounts page](
 https://console.cloud.google.com/iam-admin/serviceaccounts?project=mapping-test-data)
 as `gd-earthengine-test-user@givedirectly.org` and generating a new key for
-`firebase-adminsdk-j6emn@mapping-test-data.iam.gserviceaccount.com`. This is needed
-for running tests. Remember to startup a new terminal window for changes to
+`firebase-adminsdk-j6emn@mapping-test-data.iam.gserviceaccount.com`. 
+
+* Save the file locally somewhere *outside* of your local download of this repository.
+
+* Set `export GOOGLE_APPLICATION_CREDENTIALS=/path/to/secret.json` in your shell 
+(most likely in your `~/.bashrc` on linux, `~/.bash_profile` on mac). Remember to startup a new terminal window for changes to
 take effect.
 
-* Install [Chromium](https://www.chromium.org) if on Linux. Install via the
-  usual `apt-get`-style channels. This should be relatively straightforward on
-  most systems, but is difficult/impossible on Google-internal machines. If you
-  have difficulty, you can [download a latest
-  version](https://download-chromium.appspot.com) and unzip it. Then create a
-  link, via `sudo ln -s /path/to/file/in/extracted/zip/named/chrome-wrapper
-  /usr/bin/chromium` so that Cypress knows how to find it easily.
+* Start the local web server as described above. 
 
 * Test changes locally.
 
     ```shell
-    yarn run cypress run # --browser chromium ## (only needed on Linux) 
+    yarn run cypress run # --browser chrome ## (only needed on Linux) 
     ```
 
-* Travis CI runs on each push to Github, unless the commit message has
+* *[Optional]* Travis CI runs on each push to Github, unless the commit message has
 '[skip ci]' inside it. You can trigger a run manually from the [Travis main
 page](https://travis-ci.com/givedirectly/Google-Partnership). Runs are recorded
 to the [Cypress dashboard](https://www.cypress.io/dashboard/).
 
-* Save and share your personal test results in a web interface using
-[Cypress dashboard](https://www.cypress.io/dashboard/). The dashboard also
-provides easy access to output files, footage of the test being run, etc. To find
-the record key for this project go to the Google-Partnership project at
-https://dashboard.cypress.io in the "Runs" tab under "Settings".
+* *[Optional]* Save and share your personal test results in a web interface using
```

Comment:
I think maybe we can drop this? No real reason to put non-Travis results in the dashboard, right?

Reply:
Should still link to the dashboard somewhere since it's nice. Let me know what you think of revised.